recipes/gappa/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: gappa
+  version: "0.0.0_dev"
```

Comment:
Technically, we don't publish development versions. It get's too many too easily.Think you can just call this 0.0.1?

Reply:
Also, from the tag name, it appears to be 0.0.0-dev.3? If it's ready for audience, call it a release. Use `semver` if you are new to releasing software - it may sound weird to increase the first digit that often, but in the end it's just a number. The point is to know whether changes between releases require changes to downstream tools (like a pipeline e.g. relying on the command line parameters to remain stable), and to be able to know which state of your tool was used for a particular analysis 10 years from now.